### PR TITLE
[271] jgrss/band res 

### PR DESCRIPTION
**Pull request by jgrss**
## What is this PR changing?
Following #271, this PR adds a check for coordinate alignment and a warning when concatenation inputs are not aligned.

### CODE REVIEW

File: tests/test_open.py [hunk 1]
```diff
@@ -57,7 +73,9 @@ def test_nodata(self):
             self.assertEqual(src.gw.nodataval, 0)
 
     def test_open_multiple(self):
-        with gw.open([l8_224078_20200518, l8_224078_20200518], stack_dim='time') as src:
```
Comment (jgrss) [inline tests/test_open.py:60]: Everything here and below is `black` formatting.

**Review by mmann1123 — APPROVED** [review]
Great.

**Comment by mmann1123** [comment]
@jgrss did this get stalled somehow?

**Comment by jgrss** [comment]
> @jgrss did this get stalled somehow?

I guess so -- just fell off my radar.

**Pull request merged**